### ISSUE TITLE
close-toast does not have discernible button text

### ISSUE THREAD

**Issue by jssuttles** [aws-amplify/amplify-ui#2284]
### Before creating a new issue, please confirm:

- [X] I have [searched for duplicate or closed issues](https://github.com/aws-amplify/amplify-ui/issues?q=is%3Aissue+) and [discussions](https://github.com/aws-amplify/amplify-ui/discussions).
- [X] I have tried disabling all browser extensions or using a different browser
- [X] I have tried deleting the node_modules folder and reinstalling my dependencies
- [X] I have read the guide for [submitting bug reports](https://github.com/aws-amplify/amplify-ui/blob/main/CONTRIBUTING.md#bug-reports).

### On which framework/platform are you having an issue?

Vue

### Which UI component?

Authenticator

### How is your app built?

vue cli

### What browsers are you seeing the problem on?

Chrome

### Please describe your bug.

<img width="375" alt="image" src="https://user-images.githubusercontent.com/8527741/178573036-eef248f2-7fd4-4681-8f7b-4a1e8a604e3e.png">
Button does not have discernible button text that would indicate to a screen-reader what this button is for.

### What's the expected behaviour?

The axe test passes.

### Help us reproduce the bug!

Add aria-label?
https://dequeuniversity.com/rules/axe/4.4/button-name?application=AxeChrome

### Code Snippet

```typescript
// Put your code below this line.

```

**Comment by hbuchel** [comment]
Hi @jssuttles! Thanks for opening this issue. We are indeed missing a mechanism for adding an accessible label to the Alert primitives. We have similar work noted for the React package https://github.com/aws-amplify/amplify-ui/issues/2125 to fix this issue there, but will make sure a similar fix is applied for the Vue/Angular packages.

**Comment by ErikCH** [comment]
Hi @jssuttles ! I fixed this in Angular and Vue and if you use the latest version it should have the correct accessible label. If you have any other problems, please let me know, or you can reopen this issue.

**Comment by jssuttles** [comment]
Would it be possible to get another v1 build for Vue 2?

**Comment by ErikCH** [comment]
Hi! 

Right now we are only updating V1 release for security issues. And since Vue 2 is in maintenance mode we're recommending our customers to upgrade to Vue 3, if they want to use the latest Authenticator component. Sorry about that!
